Common subdirectories: blktrace-1.0.1/btreplay and blktrace-1.0.1-fixed/btreplay
Common subdirectories: blktrace-1.0.1/btt and blktrace-1.0.1-fixed/btt
Common subdirectories: blktrace-1.0.1/doc and blktrace-1.0.1-fixed/doc
diff -u blktrace-1.0.1/Makefile blktrace-1.0.1-fixed/Makefile
--- blktrace-1.0.1/Makefile	2013-05-20 14:10:10.048911161 -0700
+++ blktrace-1.0.1-fixed/Makefile	2013-05-20 14:11:45.050011030 -0700
@@ -1,6 +1,7 @@
 CC	= gcc
 CFLAGS	= -Wall -O2 -g -W
 ALL_CFLAGS = $(CFLAGS) -D_GNU_SOURCE -D_LARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64
+LDFLAGS := -lpthread
 PROGS	= blkparse blktrace verify_blkparse blkrawverify blkiomon
 LIBS	= -lpthread
 SCRIPTS	= btrace
